core/tests/protractor_utils/LearnerDashboardPage.js

```diff
@@ -137,12 +146,10 @@ var LearnerDashboardPage = function() {
     });
   };
 
-  this.expectTitleOfExplorationSummaryTileToMatch = function(title) {
-    var explorationTitle = element(
-      by.cssContainingText('.protractor-test-exp-summary-tile-title', title));
-    waitFor.visibilityOf(
-      explorationTitle, 'Unable to find exploration ' + title);
-    expect(explorationTitle.isDisplayed()).toBe(true);
+  this.expectTitleOfExplorationSummaryTileToMatch = function(title, index) {
```

Comment:
Can we just pass the title and loop over all the tiles and select the one which matches the title? (That will make the function more easily usable, right?)

Reply:
Done